### DISCUSSION TITLE
Glass material not displayed

### DISCUSSION THREAD

**Discussion by adrianposteuca** [google/model-viewer#3764] [Q&A] [answered]
Hello all,

Hope you can help me with a problem with **glass material** for a 3d object

The model of a window is created in cinema4d and in render and preview looks good. Glass looks glass!

![C4D capure glass](https://user-images.githubusercontent.com/17274973/187929347-7b76e2b3-03fd-4823-8759-7f9922c70b0b.PNG)
Exported as a GLB file and tested in ModelViwer has the same characteristics
Glass is glass.

![MDW_window_glass](https://user-images.githubusercontent.com/17274973/187929723-04e1774a-d58f-4a42-a329-3185cd6cc1d1.PNG)

The problematic part is when I try to use it as a model in Glitch for client development there is no glass material just plain white.
You can see it here in scene viewer but also in AR

https://glitch.com/edit/#!/lems-window-test

What is wrong, what am I doing wrong? Please help :)

**Comment by elalish** [comment, marked as answer]
This is a bug I already fixed but hasn't made it into a release yet (the most common reason you'll see a difference with our editor, since it is served from master). It's incorrectly identifying your glass as a baked shadow (see `hasBakedShadow`). A quick workaround is `shadow-intensity="0"`. We'll have v2.0 out in a week or two with this fix and a lot more.

**Reply by adrianposteuca** [reply]
Thanks,

the quick fix solved the problem <shadow-intensity = "0" > , but with no shadow.
I hope I can bring back the shadows once you will released the V2.0 :)

**Comment by adrianposteuca** [comment]
Hi @elalish,

As you suggested I set up the shadow intensity at "0" and in the scene viewer it works, BUT in AR mode (webxr) there is no glass.

Is there a way to fix this in AR also?

Scene viewer

![Screenshot_20220906-155011_Chrome](https://user-images.githubusercontent.com/17274973/188640406-6c3bb5d6-5815-4fd9-b156-81f2c52bfbac.jpg)

AR scene

![Screenshot_20220906-155044_Chrome](https://user-images.githubusercontent.com/17274973/188640528-b126ff24-0a54-4d72-8ffb-4f1e91b0c6e7.jpg)

**Reply by elalish** [reply]
Oh, right. Well, the good news is we're putting out v2.0 very shortly with this fix and many others. You can even try it out now with `https://www.unpkg.com/browse/@google/model-viewer@2.0.0-rc1/`
